docs/getting_started.md

```diff
@@ -12,16 +12,16 @@ The required versions of MMCV and MMDetection for different versions of MMDetect
 
 | MMDetection3D version | MMDetection version |    MMCV version     |
 |:-------------------:|:-------------------:|:-------------------:|
-| master              | mmdet>=2.10.0, <=2.11.0| mmcv-full>=1.2.4, <=1.4|
-| 0.13.0              | mmdet>=2.10.0, <=2.11.0| mmcv-full>=1.2.4, <=1.4|
-| 0.12.0              | mmdet>=2.5.0, <=2.11.0 | mmcv-full>=1.2.4, <=1.4|
-| 0.11.0              | mmdet>=2.5.0, <=2.11.0 | mmcv-full>=1.2.4, <=1.4|
-| 0.10.0              | mmdet>=2.5.0, <=2.11.0 | mmcv-full>=1.2.4, <=1.4|
-| 0.9.0               | mmdet>=2.5.0, <=2.11.0 | mmcv-full>=1.2.4, <=1.4|
-| 0.8.0               | mmdet>=2.5.0, <=2.11.0 | mmcv-full>=1.1.5, <=1.4|
-| 0.7.0               | mmdet>=2.5.0, <=2.11.0 | mmcv-full>=1.1.5, <=1.4|
-| 0.6.0               | mmdet>=2.4.0, <=2.11.0 | mmcv-full>=1.1.3, <=1.2|
-| 0.5.0               | 2.3.0                  | mmcv-full==1.0.5|
+| master              | mmdet>=2.10.0       | mmcv-full>=1.3.2, <=1.4|
+| 0.13.0              | mmdet>=2.10.0       | mmcv-full>=1.2.4, <=1.4|
```

Comment:
We should not change the max version as the code before this PR cannot run on mmdet>=2.11.0

Reply:
Yes, we should only remove the max_version requirement of master branch.

Reply:
but if we merge this PR, we can only run on mmdet>=2.12.0